sklearn/gaussian_process/_gpr.py

```diff
@@ -391,7 +391,7 @@ def sample_y(self, X, n_samples=1, random_state=0):
         n_samples : int, default=1
             The number of samples drawn from the Gaussian process
 
-        random_state : int, RandomState, default=0
+        random_state : int, RandomState instance, default=0
```

Comment:
Should None be added as an option here?

Reply:
In fact, I think that `None` should be the default option (but let us wait for a core-developer).

Reply:
Yes `None` should be added. This would be a valid parameter which will be validated in the `check_random_state`.
Here, I assume that we want to have a deterministic behaviour by default which is different from the other case. But we could still decide to not care and pass `None`.

Reply:
Thanks for your answers. I have added None as an option but I did not change the default value.